src/bin/cargo/commands/fix.rs

```diff
@@ -72,6 +72,20 @@ pub fn cli() -> App {
                 .long("allow-staged")
                 .help("Fix code even if the working directory has staged changes"),
         )
+        .arg(
+            Arg::with_name("clippy")
+                .long("clippy")
+                .help("Get fix suggestions from clippy instead of check")
+                .hidden(true),
```

Comment:
do we want to make this hidden? We could leave it visible, it wont let people use it without the -Z unstable-options either way.

Reply:
hidden, please.